actors/builtin/reward/reward_test.go

```diff
@@ -27,6 +27,9 @@ func TestConstructor(t *testing.T) {
 			Build(t)
 		startRealizedPower := abi.NewStoragePower(0)
 		actor.constructAndVerify(rt, &startRealizedPower)
+		st := getState(rt)
+		assert.Equal(t, abi.ChainEpoch(0), st.Epoch)
+		assert.Equal(t, abi.NewStoragePower(0), st.CumsumRealized)
```

Comment:
Please assert the value of the first epoch's reward. Hardcoded is probably ok, but at least check >0.